wiki/en/Software-Manual.md

```diff
@@ -26,9 +26,11 @@ This manual documents the Jamulus client application for use by musicians and si
 	<figcaption>Your local mix when connected to a Server</figcaption>
 </figure>
 
-## Delay and Buffer LEDs
+## Ping, Delay and Jitter
 
-**Delay** shows the status of the current audio latency:
+**Ping** shows your network latency in milliseconds, the lower the better. Ping time contributes to overall delay (see below). The most probable causes for a high ping are that your distance to the server is too large or your internet connection is not sufficient.   
```

Comment:
> or your internet connection is not sufficient. 

This sounds a bit vague and might be taken to mean that increasing bandwidth will solve the problem - which it often won't.

Reply:
Yes, the wording here is the original from the "measurements" section we had but I wondered about that. Maybe just leave it as _"The most probable causes for a high ping are that your distance to the server is too large."_ Otherwise, we'll start getting into troubleshooting routers and whatnot here.

Reply:
Yes, I don't think it's worth getting into all the reasons why.